internal/adapter/http/user.go

```diff
@@ -17,6 +17,12 @@ func NewUserController(usecase interfaces.User) *UserController {
 	}
 }
 
+type PasswordResetInput struct {
+	Email    *string `json:"email"`
+	Token    *string `json:"token"`
+	Password *string `json:"password"`
```

Comment:
```suggestion
	Email    string `json:"email"`
	Token    string `json:"token"`
	Password string `json:"password"`
```

Empty string is almost same as nil